source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -361,28 +383,53 @@ export class Carousel extends Container {
 
         const prevElementDiv: HTMLElement = document.createElement("div");
         prevElementDiv.className = this.hostConfig.makeCssClassName(
-            "swiper-button-prev",
-            "ac-carousel-left"
+            "swiper-button-prev"
         );
-        containerForAdorners.appendChild(prevElementDiv);
-        Utils.addCancelSelectActionEventHandler(prevElementDiv);
 
         const nextElementDiv: HTMLElement = document.createElement("div");
         nextElementDiv.className = this.hostConfig.makeCssClassName(
-            "swiper-button-next",
-            "ac-carousel-right"
+            "swiper-button-next"
         );
-        containerForAdorners.appendChild(nextElementDiv);
-        Utils.addCancelSelectActionEventHandler(nextElementDiv);
+
+        if (this.carouselOrientation === Enums.Orientation.Horizontal) {
+            prevElementDiv.classList.add(this.hostConfig.makeCssClassName(
+                "ac-carousel-left"
+            ));
+            nextElementDiv.classList.add(this.hostConfig.makeCssClassName(
+                "ac-carousel-right"
+            ));
+        } else {
+            carouselContainer.style.height = this.verticalCarouselHeight + "px";
```

Comment:
is user input validated? have we put good safeguard against bad values?

Reply:
We validate pixel properties to be positive integers